deploy/docs/ContainerLogs.md

```diff
@@ -220,7 +221,97 @@ and `time` key will be visible in Sumo:
 }
 ```
 
-## Support for multiline logs for containerd
+## Configuration for multiline logs for containerd log format
+
+### Using old multiline parser
+
+In order to collect multiline logs in containerd log format which may contain very long lines that
+container runtime splits into partial log lines (marked by `P`) you may use prepared Fluent Bit configuration
+available in [/examples/multiline_logs/values_containerd.yaml](/examples/multiline_logs/values_containerd.yaml).
+
+This configuration uses [old multiline configuration parameters][old-multiline-parser] with
+[lua filter][lua-filter] to join multipart logs (taking into account log tags `P` and `F`)
+and remove parts specific for containerd format and not related to the application log.
+
+To adjust multiline parser to your logs please modify parser in  `customParsers` section to match
+the first line of multiline log:
+
+```yaml
+    customParsers: |
+      [PARSER]
+          Name         containerd_multiline_pattern
+          Format       regex
+          Regex        (?<time>^\d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2}[.]\d+Z) (?<stream>stdout|stderr) (?<logtag>[P|F]) (?<log>\w+ \d+ \d{2}:\d{2}:\d{2} .*)
+          Time_Key     time
+          Time_Format  %Y-%m-%dT%H:%M:%S.%LZ
+```
+
+**_Notice:_** Strict regular expression is needed to differentiate between the first line, next line of the log
+and partial log line. When you know that your logs will not contain very long lines you may prepare regular expression
+in generic form, e.g. `(?<time>^\d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2}[.]\d+Z) (?<stream>stdout|stderr) (?<logtag>[P|F]) (?<log>\S+ .*)`
+
+To test the regular expression, you can may use the [Rubular][rubular] web site.
```

Comment:
`may` sounds strange in this context, as if we're giving permission. 😄 
```suggestion
To test the regular expression, you can can use the [Rubular][rubular] web site.
```

Reply:
I think that one `can` is enough 😄